projects/netgrif-components/src/lib/navigation/navigation-double-drawer/navigation-double-drawer.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="logo" [ngClass]="{'logo-closed': !configMenu2.opened}" fxLayoutAlign="start center" fxHide.lt-md>
+    <button mat-icon-button (click)="toggleMenu2()" class="mat-button">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <img *ngIf="configMenu2.opened && image !== undefined" #logo [src]="image" class="logoimg"
+         [routerLink]="imageRouterLink !== undefined ? imageRouterLink : []">
+</div>
+<mat-toolbar fxHide.gt-sm class="toolbar">
+    <mat-toolbar-row fxLayout="row" fxLayoutAlign="space-between stretch">
+        <div fxFlex fxLayoutAlign="start center">
+            <button mat-icon-button (click)="toggleMenu()">
+                <mat-icon>menu</mat-icon>
+            </button>
+        </div>
+        <div class="logo-tool" fxLayoutAlign="center center" fxFlex >
+            <img #logo [src]="image" class="logoimg-tool"
```

Comment:
alt? 
The required alt attribute specifies an alternate text for an image, if the image cannot be displayed.

Reply:
done